### PR TITLE
added tag to roles navigation to make it extensible

### PR DESCRIPTION
**Pull request by massivesmo**

### CODE REVIEW

File: src/Sulu/Bundle/SecurityBundle/DependencyInjection/Compiler/AddContentNavigationPass.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+use Sulu\Bundle\AdminBundle\DependencyInjection\Compiler\ContentNavigationPass;
+
+/**
+ * Add all services with the tag "sulu.contact.content_navigation" to the content navigation
+ *
+ * @package Sulu\Bundle\AdminBundle\DependencyInjection\Compiler
+ */
+class AddContentNavigationPass extends ContentNavigationPass
+{
+
```
Comment (dantleech) [inline src/Sulu/Bundle/SecurityBundle/DependencyInjection/Compiler/AddContentNavigationPass.php]: Extra line

File: src/Sulu/Bundle/SecurityBundle/DependencyInjection/Compiler/AddContentNavigationPass.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\SecurityBundle\DependencyInjection\Compiler;
+
+use Sulu\Bundle\AdminBundle\DependencyInjection\Compiler\ContentNavigationPass;
+
+/**
+ * Add all services with the tag "sulu.contact.content_navigation" to the content navigation
```
Comment (dantleech) [inline src/Sulu/Bundle/SecurityBundle/DependencyInjection/Compiler/AddContentNavigationPass.php]: Wrong doc

File: src/Sulu/Bundle/SecurityBundle/DependencyInjection/Compiler/AddContentNavigationPass.php [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ *
+ * @package Sulu\Bundle\AdminBundle\DependencyInjection\Compiler
```
Comment (dantleech) [inline src/Sulu/Bundle/SecurityBundle/DependencyInjection/Compiler/AddContentNavigationPass.php]: Remove `@package`

File: src/Sulu/Bundle/SecurityBundle/DependencyInjection/Compiler/AddContentNavigationPass.php [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+ */
+class AddContentNavigationPass extends ContentNavigationPass
+{
+
+    public function __construct()
+    {
+        $this->tag = 'sulu.security.admin.roles_navigation';
+        $this->serviceName = 'sulu_security.admin.roles_navigation';
+    }
+
```
Comment (dantleech) [inline src/Sulu/Bundle/SecurityBundle/DependencyInjection/Compiler/AddContentNavigationPass.php]: Extra line

**Comment by massivesmo** [comment]
everything ok now?

**Comment by wachterjohannes** [comment]
@dantleech looks good for me!

**Comment by dantleech** [comment]
ok .. @massivesmo can you rebase and then I will merge.

**Comment by massivesmo** [comment]
@dantleech there was only a small conflict with the changelog.
everything else is untouched

**Comment by dantleech** [comment]
There is always a conflict in the CHANGELOG unless you are lucky enough to not have had any merges since you made your PR :)

**Pull request merged**